docs/components/connectors/use-connectors.md

```diff
@@ -97,6 +97,70 @@ For further details on how secrets are implemented in Connectors, consult our [C
 Using this in other areas can lead to unexpected results and incidents.
 :::
 
+## Response mapping
+
+Some connectors have a `Response Mapping` section that typically consists of two fields: (1) `Result Variable` and (2) `Result Expression`.
+These fields are used to export response from an external connector call into process variables.
+
+### Result Variable
+
+This field declares a single process variable to export response from a connector call. You are able to use this process variable further in the process.
+
+### Result Expression
+
+This field allows to map a connector response into multiple process variable which you are able to use further in the process. You can also transform the extracted values using [FEEL expressions](../concepts/expressions.md).
+Please, keep in mind, while using this field, a process variable with name `response` is reserved.
+
+### Example
+
+Given your connector makes an external call to an arbitrary weather service. The weather service returns the following response:
+
+```json
+{
+  "status": 200,
+  "headers": {
+    "date": "Thu, 19 Jan 2023 14:02:29 GMT",
+    "transfer-encoding": "chunked",
+    "content-type": "application/json; charset=utf-8",
+    "connection": "keep-alive"
+  },
+  "body": {
+    "latitude": 52.52,
+    "longitude": 13.4,
+    "generationtime_ms": 0.22804737091064453,
+    "utc_offset_seconds": 0,
+    "timezone": "GMT",
+    "timezone_abbreviation": "GMT",
+    "elevation": 45.0,
+    "current_weather": {
+      "temperature": 1.0,
+      "windspeed": 10.1,
+      "winddirection": 186.0,
+      "weathercode": 2,
+      "time": "2023-01-19T14:00"
+    }
+  }
+}
+```
+
+If you declare a variable `myWeatherResponse` in the `Result Variable` field, the whole response will be mapped to the declared variable.
+
+Now, let's imagine that you wish to extract only temperature into a process variable `berlinWeather` and wind speed into `berlinWindSpeed`. Let's also imagine, you need weather in Fahrenheit declared in `berlinWeatherInFahrenheit`.
+
+In that case, you could declare `Result Expression` as follows:
```

Comment:
maybe we can add a sentence that only the values defined in the result expression are stored in the result variable "myWeatherResponse". Just to make it explicit.

Reply:
maybe as last sentence under the response mapping result image

Reply:
Sorry, didn't get your comment. The variable `myWeatherResponse` will store everything and it's not related to result expression.